pkg/networkpath/traceroute/runner.go

```diff
@@ -117,10 +117,10 @@ func (r *Runner) RunTraceroute(ctx context.Context, cfg Config) (payload.Network
 	}
 
 	var timeout time.Duration
-	if cfg.TimeoutMs == 0 {
+	if cfg.Timeout == 0 {
```

Comment:
We might want to remove this and favor having it come in from the config everywhere